src/components/HomeWelcome/index.jsx

```diff
@@ -0,0 +1,33 @@
+import Smilies from "assets/icons/smilies.svg";
+import Description from "components/shared/Description";
+import PrimaryParagraph from "components/shared/Paragraph";
+import Title from "components/shared/Title";
+import styles from "./index.module.scss";
+import Dot from "assets/icons/oval.svg";
+import Button from "components/shared/Button";
+
+const HomeWelcome = () => {
+  return (
```

Comment:
you can omit the curly braces and return statement when the function only do a return, so in this case:

```suggestion
const HomeWelcome = () => (
```

remember to remove the curly braces from the bottom